src/Components/DonationFormModal.js

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import '../Styles/DonationFormModal.scss';
+
+const DonationFormModal = () => {
+  const [formState, setFormState] = useState({
+    firstName: '',
+    lastName: '',
+    ownerStory: '',
+    petName: '',
+    age: '',
+    type: '',
+    gender: '',
+    petDescription: '',
+    imageUpload: ''
+  })
+  return (
+    <div className="modal">
+      <form className="donation-form" onSubmit={(e) => e.preventDefault()}>
+        <h2 className="form-title">Donation Form</h2>
+        <section className="owner-info-container">
+          <h3 className="form-subtitle">Owner Information</h3>
+          <div className="owner-name-container">
+            <div className="input-label">
+              <label htmlFor="firstName">First Name</label>
+              <input
+                type="text"
+                value={formState.firstName}
+                id="firstName"
+                placeholder="e.g. Samantha"
+                onChange={(e) =>
```

Comment:
Great job using hooks to set the state immediately as the user is typing, using hooks.